Source/WebInspectorUI/UserInterface/Models/FontStyles.js

```diff
@@ -304,15 +308,17 @@ WI.FontStyles = class FontStyles
         for (let fontFeatureSetting of features) {
             let featureSettingValue = style.featuresMap.get(fontFeatureSetting);
             if (featureSettingValue || featureSettingValue === 0) {
-                resultProperties.features ??= new Map;
-                resultProperties.features.set(fontFeatureSetting, featureSettingValue);
+                resultProperty.features ??= new Map;
+                resultProperty.features.set(fontFeatureSetting, featureSettingValue);
 
                 // Remove the tag so it is not presented twice.
                 style.featuresMap.delete(fontFeatureSetting);
             }
         }
 
-        return resultProperties;
+        resultProperty.isImportant = style.domNodeStyle.effectivePropertyForName(name)?.important;
```

Comment:
FYI we're already calling this inside `_effectivePropertyValueForName`, so i think we could/should avoid that
```js
let effectivePropertyForName = style.domNodeStyle.effectivePropertyForName(name);

let value = effectivePropertyForName?.value;

...

resultProperty.isImportant = effectivePropertyForName?.important ?? false;

```

Reply:
Added

Reply:
We need to keep the return resultProperties; statement included.